x-pack/legacy/plugins/uptime/server/lib/requests/get_states_index_status.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { UMElasticsearchQueryFn } from '../adapters';
+import { StatesIndexStatus } from '../../../common/graphql/types';
+import { INDEX_NAMES } from '../../../common/constants';
+
+export const getStatesIndexStatus: UMElasticsearchQueryFn<{}, StatesIndexStatus> = async ({
+  callES,
+}) => {
+  const {
+    _shards: { total },
+    count,
+  } = await callES('count', { index: INDEX_NAMES.HEARTBEAT });
+  return {
+    indexExists: total > 0,
+    docCount: {
+      count,
+    },
+  };
+};
```

Comment:
This looks identical to getDocCount...

Reply:
You are correct. They are both in use. We should delete `getDocCount` as it's touching the least amount of code, and replace it with a call to `getStatesIndexStatus`, which we should rename `getIndexStatus`. The reason this code exists is that it is leftover from a previous feature that we built out and subsequently deleted when we added the `iterator` logic for building our monitors list.

Reply:
I created https://github.com/elastic/kibana/issues/56864 to address this.